src/App.js

```diff
@@ -1,86 +1,84 @@
-import logo from './logo.svg';
 import './App.css';
 import React from 'react';
 import { useState } from 'react';
 import Food from './Food';
 
 
-
-
-
-
-
-
 function App() {
   const [foodList, setFoodList] = useState([]);
+  const [likeCount, setLikeCount] = useState([]);
   const [inputFood, setInputFood] = useState("");
 
   const [isSingleView, setIsSingleView] = useState(false);
   const [currIndex, setCurrIndex] = useState(0);
 
   const addFood = () => {
     setFoodList([...foodList, inputFood]);
-    setLikes([...likes, 0]);
     setInputFood("");
+    setLikeCount([...likeCount, 0]);
   }
 
   const switchInput = () => {
     setIsSingleView(!isSingleView);
   }
 
-  const nextFood = () => {
-    if (currIndex < foodList.length - 1) {
-      setCurrIndex(currIndex + 1);
-    } else {
-      setCurrIndex(0);
-    }
-  }
-
-
-  const [likes, setLikes] = useState([]);
   // true = single view?
   return (
     <div className="App">
       <header className="App-header">
         <div>
           {
-            <button onClick={switchInput}> {
+            <button className="float-switch-button my-button" onClick={switchInput}> {
               isSingleView ? "View List" : "View Single"
             } </button>
           }
-          <h2> Add a New Food</h2>
-          <input type="text" onChange={(event) => { setInputFood(event.target.value) }} value={inputFood} />
-          <div>
-            <button onClick={addFood}>  Submit </button>
-          </div>
-        </div>
-        <div>
-          {isSingleView && (
+          <div className="grid-container">
             <div>
-              <Food
-                name={foodList[currIndex]}
-                index={currIndex}
-                foodList={foodList}
-                setFoodList={setFoodList}
-              >
-              </Food>
-              <button onClick={nextFood}> Pass Me </button>
+              <h2 id="main-title"> Add a New Food</h2>
+              <input className="input-field" type="text" onChange={(event) => { setInputFood(event.target.value) }} value={inputFood} />
+              <div>
+                <button className="my-button" onClick={addFood}>  Submit </button>
+              </div>
             </div>
-          )
-          }
-          {!isSingleView &&
-            foodList.map((value, index) => {
-              return <Food
-                name={value}
-                index={index}
-                foodList={foodList}
-                setFoodList={setFoodList}
-                likes={likes}
-                setLikes={setLikes}
-              >
-              </Food>
-            })
-          }
+
+            <div>
+              { foodList.length < 1 &&
```

Comment:
super nit: i would change it to this so that it's easier for others to read that it is the negative of eg line 48!
```suggestion
              { foodList.length <= 0 &&
```